D/Display/DisplayController.cs

```diff
@@ -171,7 +171,7 @@ private void HorizontalRetraceCallback(ulong skewNsec, object context)
                 {
                     // Normal line : 32 bits of border pattern, 1024 bits of display, 32 bits of border pattern
                     // Border pattern: low byte on lines 4n, 4n+1; high byte on 4n+2, 4n+3.
-                    int patternByte = (effectiveScanline & 0x2) == 0 ? _displayBorder & 0xff : _displayBorder >> 8;
+                    int patternByte = (effectiveScanline & 0x2) == 2 ? _displayBorder & 0xff : _displayBorder >> 8;
```

Comment:
The bug here is more subtle -- changing this here provides the correct pattern alignment for Viewpoint, but breaks the pattern on Interlisp-D.  (Currently Interlisp-D looks fine, VP is broken).  Prefer not changing this now until the root cause can be determined.